src/Command/Task/RunCommand.php

```diff
@@ -2,26 +2,32 @@
 
 namespace Glooby\TaskBundle\Command\Task;
 
+use Doctrine\ORM\EntityManagerInterface;
 use Doctrine\ORM\NoResultException;
 use Glooby\TaskBundle\Task\TaskRunner;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
-use Symfony\Component\DependencyInjection\ContainerInterface;
 
 /**
  * @author Emil Kilhage
  */
 class RunCommand extends Command
 {
     protected static $defaultName = 'task:run';
-    private $container;
 
-    public function __construct(ContainerInterface $container){
+    /** @var EntityManagerInterface */
+    private $em;
+
+    /** @var TaskRunner */
+    private $runner;
+
+    public function __construct(EntityManagerInterface $em, TaskRunner $runner){
```

Comment:
Could you update `Resources/config/services.yml` to point to the new injected services? Otherwise running `bin/console` commands break when used in a Symfony 4 project.

For instance
``` glooby_task.task_run_command:
        class:
            Glooby\TaskBundle\Command\Task\RunCommand
        arguments:
            - '@doctrine.orm.entity_manager'
            - '@glooby_task.task_runner'
        tags:
            - { name: 'console.command', command: 'task:run' }

Reply:
sure I can but maybe we can use:
```
services:
    _defaults:
        autowire: true
        autoconfigure: true
        public: true
```

functionality & refactor all other services removing them setters and inject in constructor all required dependencies ?
i.e. QueuePruner